src/Internal/ObjectExporter/ClosureExporter.php

```diff
@@ -190,7 +226,9 @@ private function closureHandleUses(
     ) : void {
         if (! $this->exporter->closureSnapshotUses) {
             throw new ExportException(
-                "The closure has bound variables through 'use', this is not supported by default. " .
+                "The closure has bound variables" .
+                    ($closure->hasAttribute('arrow_function') ? "" : " through 'use'") .
+                    ", this is not supported by default. " .
```

Comment:
Could you please say instead, depending on the type:

- The closure has bound variables through 'use'
- The arrow function uses variables in the parent scope